index.test.js

```diff
@@ -8,53 +8,183 @@ describe('[Exercise 1] trimProperties', () => {
+    expect(secondSeason).toBe('fall')
+  })
+
+  test('[11] the THIRD call of seasons.next returns "winter"', () => {
+    const firstSeason = seasons.next()
+    const secondSeason = seasons.next()
+    const thirdSeason = seasons.next()
+    expect(thirdSeason).toBe('winter')
+  })
+
+  test('[12] the FOURTH call of seasons.next returns "spring"', () => {
+    const firstSeason = seasons.next()
+    const secondSeason = seasons.next()
+    const thirdSeason = seasons.next()
+    const fourthSeason = seasons.next()
+    expect(fourthSeason).toBe('spring')
+  })
+
+  test('[13] the FIFTH call of seasons.next returns again "summer"', () => {
+    for(let i = 0; i < 4; i++){
+      seasons.next()
+    }
+    const fifthSeason = seasons.next()
+    expect(fifthSeason).toBe('summer')
+  })
+
+  test('[14] the 40th call of seasons.next returns "spring"', () => {
+    for(let i = 0; i < 39; i++){
+      seasons.next()
+    }
+    const fortySeason = seasons.next()
+    expect(fortySeason).toBe('spring')
+  })
 })
 
 describe('[Exercise 6] Car', () => {
   let focus
   beforeEach(() => {
     focus = new utils.Car('focus', 20, 30) // each test must start with a fresh car
   })
-  // test('[15] driving the car returns the updated odometer', () => {})
-  // test('[16] driving the car uses gas', () => {})
-  // test('[17] refueling allows to keep driving', () => {})
-  // test('[18] adding fuel to a full tank has no effect', () => {})
+
+  test('[15A] car is defined', () => {
+    expect(utils.Car).toBeDefined()
+    expect(utils.Car).toBeInstanceOf(Function)
+  })
+
+  test('[15B] driving the car returns the updated odometer', () => {
+    focus.drive(19)
+    expect(focus.odometer).toBe(19)
+    focus.drive(11)
+    expect(focus.odometer).toBe(30)
+  })
+
+  test('[16] driving the car uses gas', () => {
+    focus.drive(10)
+    expect(focus.tank).toBeLessThan(20)
+  })
+
+  test('[17] refueling allows to keep driving', () => {
+    focus.drive(600)
+    expect(focus.tank).toBe(0)
+    focus.refuel(15)
+    expect(focus.tank).toBe(15)
+  })
+
+  test('[18] adding fuel to a full tank has no effect', () => {
+    expect(focus.tank).toBe(20)
+    focus.refuel(10)
+    expect(focus.tank).not.toBeGreaterThan(20)
+    expect(focus.tank).toBe(20)
+  })
 })
 
 describe('[Exercise 7] isEvenNumberAsync', () => {
-  // test('[19] resolves true if passed an even number', () => {})
-  // test('[20] resolves false if passed an odd number', () => {})
+  test('[19] resolves true if passed an even number', async () => {
+    const evenNum = await utils.isEvenNumberAsync(6)
+      .then(res => {
+        return res
+      })
+      .catch(err => {
+        return err
+      })
+
+    expect(evenNum).toBe(true)
+    expect(evenNum).not.toBe(false)
+    expect(evenNum).toBeTruthy()
+  })
+
+  test('[20] resolves false if passed an odd number', async () => {
+    const oddNum = await utils.isEvenNumberAsync(9)
+    expect(oddNum).not.toBe(true)
```

Comment:
You don't need all 3 expects. If you check that it's false that is enough for this test